conf/mig-agent-conf.go.inc

```diff
@@ -156,3 +156,6 @@ MIIEpAIBAAKCAQEAvJQqCjE4I63S3kR9KV0EG9e/lX/bZxa/2QVvZGi9/Suj65nD
 ........
 RMSEpg+wuIVnKUi6KThiMKyXfZaTX7BDuR/ezE/JHs1TN5Hkw43TCQ==
 -----END RSA PRIVATE KEY-----`)
+
+//PERSISTENTMODULES contains module names and params that are to be started when the agent starts initially
+var PERSISTENTMODULES = [][]string {{"audit",`{"class":"parameters", "parameters":{"rulefilepath": "audit.rules.json", "outputsockets": ["/path/to/unix.sock"], "serveraddress": "mozdef_url"}}`}}
```

Comment:
Currently, I don't decide on which option is to be used, as we are not writing to the unix socket yet and are dispatching events to url only. Though I think the behaviour can be modelled so that if both option are provided we write to both server and socket (does that make sense ?). However the option to specify an output mode and then providing the particular output address , seems a better option to me. (currently the code doesn't have this)